### PR TITLE
sign in sign up by devise

### PR DESCRIPTION
**Pull request by ngtrduc**

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<h2><%= t :login %></h2>
+<div class="col-md-6 col-md-offset-2">
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```
Comment (daonhat) [inline app/views/devise/sessions/new.html.erb]: ident

File: app/views/devise/shared/_links.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<%- if controller_name != 'sessions' %>
```
Comment (daonhat) [inline app/views/devise/shared/_links.html.erb]: dùng "

**Comment by daonhat** [comment]
ok

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<h2><%= t :signup %></h2>
+<div class="col-md-6 col-md-offset-2">
+  <%= form_for(resource, as: resource_name,
```
Comment (hellovietnam93) [inline app/views/devise/registrations/new.html.erb]: bỏ ()

File: app/views/devise/registrations/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+    <div class="field">
+      <%= f.label t :email %><br />
+      <%= f.email_field :email, autofocus: true, class: "form-control" %>
+    </div>
+
+    <div class="field">
+      <%= f.label t :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
```
Comment (hellovietnam93) [inline app/views/devise/registrations/new.html.erb]: ident

File: app/views/devise/sessions/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+    <% if devise_mapping.rememberable? -%>
+      <div class="field">
+        <%= f.check_box :remember_me %>
+        <%= f.label t :remember_me %>
+      </div>
+    <% end -%>
+
+    <div class="actions">
+      <%= f.submit "Log in", class: "btn btn-primary" %>
```
Comment (hellovietnam93) [inline app/views/devise/sessions/new.html.erb]: I18n

File: app/views/devise/shared/_links.html.erb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<%- if controller_name != "sessions" %>
```
Comment (hellovietnam93) [inline app/views/devise/shared/_links.html.erb]: unless

**Pull request merged**